src/main/java/com/arcbees/website/client/BootstrapperImpl.java

```diff
@@ -52,9 +55,16 @@ public void onBootstrap() {
         analytics.enablePlugin(AnalyticsPlugin.DISPLAY);
         analytics.enablePlugin(AnalyticsPlugin.ENHANCED_LINK_ATTRIBUTION);
 
+        int variationId = getVariationId();
+        experimentHolder.setVariationId(variationId);
+
         validateNameTokenLanguageAndRevealPlace();
     }
 
+    private native int getVariationId() /*-{
```

Comment:
I would create a face pour cxAPI rather than having this in here. It is a bad practice since it makes this class hard to unit test